tests/PHPStan/Rules/Properties/data/invalid-callable-property-type.php

```diff
@@ -0,0 +1,59 @@
+<?php declare(strict_types = 1);
+
+namespace InvalidCallablePropertyType;
+
+interface Foo
+{
+	public function fooBar(): void;
+}
+
+interface Bar
+{
+	public function barBaz(): void;
+}
+
+class HelloWorld
+{
+	/** @var callable(): void */
+	public callable $a;
+
+	/** @var callable(): void|null */
+	public ?callable $b;
+
+	/** @var callable(): void|string */
+	public callable|string $c;
+
+	/** @var Foo&callable(): void */
+	public callable&Foo $d;
```

Comment:
This isn't valid for a different reason that should be a separate rule: https://3v4l.org/bM6Ap#v8.2.11 (because it's not valid anywhere, not just in properties)

Reply:
Removed.